pkg/openstack/actuator.go

```diff
@@ -136,10 +140,30 @@ func (a *OpenStackActuator) sync(ctx context.Context, cr *minterv1.CredentialsRe
 	return nil
 }
 
-func (a *OpenStackActuator) prepareCloudsData(clouds string) (string, error) {
-	cloud, err := gophercloudutils.
+func (a *OpenStackActuator) prepareCloudsData(content string) (string, error) {
+	var clouds clientconfig.Clouds
 
-	return clouds, nil
+	err := yaml.Unmarshal([]byte(content), &clouds)
+	if err != nil {
+		return "", err
+	}
+
```

Comment:
Please can you add a comment linking to the bug here? Any future maintainer looking at this will see that we never write an incorrect value and may assume that this can be safely removed. Finding an original git commit message is not always easy.

Related: perhaps call the method something explicit like fixInvalidCACertFile?

Reply:
done